reactDashboard/src/components/HeatMap/Heatmap.css

```diff
@@ -2,34 +2,58 @@
   overflow-y: auto;
   max-height: 600px;
   height: 100%;
-  background-color: #ffffff;
+  background-color: white;
+}
+
+.darkmode .heatmap {
+  background-color: rgb(64, 64, 64);
+  box-shadow: none;
+}
+
+.heatmap-container {
+  padding: 0;
 }
 
 .heatmap table {
-  overflow-y: auto;
+  border-collapse: separate;
+  border-spacing: 0;
 }
 
 .heatmap table th {
   vertical-align: top;
   position: sticky;
+  position: -webkit-sticky;
   top: 0;
-  z-index: 2;
   background-color: white;
   color: #133a53;
 }
 
-.heatmap table th .subheading {
+.darkmode .heatmap table th {
+  color: #f2f2f2;
+  background-color: rgb(64, 64, 64);
+}
+
+.heatmap table th div.subheading {
   font-weight: normal;
 }
 
+.table th {
+  border-top: none;
+}
+
+.darkmode .table thead th {
+  border-bottom: 1px solid #020202;
```

Comment:
Should match the selector in the block above and only need border-bottom-color here

Reply:
Could be merged with the block at line 31

Reply:
see above

Reply:
Ok this one has only a darkmode border - can you create the border in lightmode so that you only override the colour here ?

Reply:
updated and comment added in code